### PR TITLE
#A11: As Admin, I would like an admin dashboard to manage the app #52 (PyMongo)

### PR DESCRIPTION
**Pull request by stefdworschak**
# Business Analysis 🟠 Code Institute 🟠 Community Project - Pull Request 🎉🎉

## Pull Request References:
#52

## PR contains:
- Create custom ModelView Flask Admin class to be able to use Flask PyMongo instead of Mongoengine
- Fix User model and loign/register views to set full User information on model and sign in user

### CODE REVIEW

File: app/templates/admin/admin_base.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    {% block head_css %}
+        <link href="{{ admin_static.url(filename='bootstrap/bootstrap2/swatch/{swatch}/bootstrap.min.css'.format(swatch=config.get('FLASK_ADMIN_SWATCH', 'default')), v='2.3.2') }}" rel="stylesheet">
+        <link href="{{ admin_static.url(filename='bootstrap/bootstrap2/css/bootstrap-responsive.css', v='2.3.2') }}" rel="stylesheet">
+        <link href="{{ admin_static.url(filename='admin/css/bootstrap2/admin.css', v='1.1.1') }}" rel="stylesheet">
+        {% if admin_view.extra_css %}
+          {% for css_url in admin_view.extra_css %}
+            <link href="{{ css_url }}" rel="stylesheet">
+          {% endfor %}
+        {% endif %}
+        <style>
```
Comment (auxfuse) [inline app/templates/admin/admin_base.html:23]: Is this necessary? The Frontend Developer in me is thinking it isn't and could be moved to an externally linked stylesheet.

File: app/templates/admin/admin_base.html [hunk 2]
```diff
@@ -0,0 +1,83 @@
+{% import 'admin/layout.html' as layout with context -%}
+{% import 'admin/static.html' as admin_static with context %}
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>{% block title %}{% if admin_view.category %}{{ admin_view.category }} - {% endif %}{{ admin_view.name }} - {{ admin_view.admin.name }}{% endblock %}</title>
+    {% block head_meta %}
+        <meta charset="UTF-8">
```
Comment (auxfuse) [inline app/templates/admin/admin_base.html:8]: We seem to have some weird additional indentation for the addition head child elements here.

**Review by auxfuse — CHANGES_REQUESTED** [review]
Trying to figure out how all this works together, but the logic play-by-play seems sound. Just two little notes in this, not necessarily to be changed, but moreso for discussion. :D Thanks @stefdworschak 🤜

File: app/templates/admin/admin_base.html (hunk 2 above)
Comment (stefdworschak) [reply app/templates/admin/admin_base.html:8]: Ah I copied this from Flask Admin. We can maybe redo the the whole template in time.

File: app/templates/admin/admin_base.html (hunk 1 above)
Comment (stefdworschak) [reply app/templates/admin/admin_base.html:23]: Ah I copied this from Flask Admin. We can maybe redo the the whole template in time.

Comment (auxfuse) [reply app/templates/admin/admin_base.html:23]: Ah in that case we can leave it for a future bit of work 😁😁

File: app/templates/admin/admin_base.html (hunk 2 above)
Comment (auxfuse) [reply app/templates/admin/admin_base.html:8]: We can revisit at a later date! 😁🤜

**Review by auxfuse — APPROVED** [review]
LGTM 🙌🙌

**Pull request merged**